docs/how-to/version-statuses.md

```diff
@@ -0,0 +1,30 @@
+
+
+## Route Information
+Navigate to: \
+```/org/{:organization}/{:project}/{:application}/versions```
```

Comment:
This documentation is user facing not internal, starting a document with "Route Information" is very off. I would expect it to start with something around the capabilities being talked about.